admin_guide/backup_restore.adoc

```diff
@@ -732,6 +732,39 @@ On each master:
 
 The procedure to add an etcd member is complete.
 
+[[backup-restore-etcd-snapshot]]
+== Snapshot Save and Restore
+
+Snapshot provides commands to restore a snapshot of a running etcd server into
+a fresh cluster. 'snapshot save' writes a point-in-time snapshot of the etcd
```

Comment:
Should the first sentence include the save part? _Snapshot provides commands to **save and** restore a snapshot
Should the `snapshot save` sentence be in its own paragraph like restore? 
Also, should the sentence start _Snapshot save_ (not the command), like restore?

Reply:
Fixed.